site/content/contribute/server/feature-flags.md

```diff
@@ -26,13 +26,115 @@ Feature flags allow us to be more confident in shipping features continuously to
+
+ 1. Find your split and click on it.
+ 2. Under the name of the split at the top, select the **Environment** that you want to target.
+     - See [Environments](#Environments) for a description of each.
+ 3. In the center of the screen, select **Add Rules**.
+ 4. Define your treatments:
+     - Recall that treatments are just the values your feature flag can have.
+     - Boolean flags should use the default "on" and "off" treatments.
+     - For non-boolean flags, change the treatments to be the possible values. E.g. for a split that specifies a plugin's version you could have three treatments of "1.0", "1.1" and "2.0".
+ 5. Down the page a bit, set the default rule to one of your treatments.
+     - This will set the flag for all servers in the environment while you don't have any other targeting rules configured.
+ 6. At the top right, click **Save changes** and confirm them on the next screen.
+
+That covers targeting an environment with a split and adding a default rule for all servers in that environment. The next section covers targeting specific servers.
+
+### Target a specific server
+
+The ID for a "server" as seen by Split is the Mattermost server's telemetry ID. To target a specific server you need that telemetry ID. The main way to get the telemetry ID from a server would be from the analytics but you can also manually grab the telemetry ID by one of the following methods:
+
+#### Get telemetry ID via inspecting browser traffic
+
+ 1. In Chrome (any browser with dev tools will work but instructions are specific to Chrome), open the developer console and switch to the Network tab.
+ 2. Type "config" into the filter for the network requests.
+ 3. Go to the server's URL (e.g. xxxxxx.test.mattermost.cloud).
+ 4. In the network tab, click on the request for `/api/v4/config/client?format=old`.
+ 5. Click on the "Response" tab and search for "TelemetryId".
+ 6. Copy the value of "TelemetryId".
+
+#### Get telemetry ID via curl and jq
+
+```bash
+$ curl -s https://<your-workspace-url>/api/v4/config/client?format=old | jq ".TelemetryId"
+```
+#### Target using the telemetry ID
+
+With the telemetry ID, go to your split and do the following:
+
+ 1. Find the section "Create whitelists" and click **Add whitelist**.
+ 2. Choose the treatment value you want the workspace to get.
+ 3. Under "Server" paste in your telemetry ID.
+ 4. At the top right, click **Save changes** and confirm them on the next screen.
+
+Now your workspace will have its feature flag set according to this rule, regardless of what the default for the split is set to.
+
+### Add or update a feature flag for community.mattermost.com
+
+Note that this will set the flag for community.mattermost.com, community-release.mattermost.com, and community-daily.mattermost.com. It is not possible to set a rule for only one.
```

Comment:
I think there's a strong use case to allow to distinguish between those a bit more. I'd like to use community-daily for testing of more "unfinished but functional" features (feature flag on), and leave community stable with the feature flag off. Would it be possible to split these three out?

Not a comment specific to the doc, but to the setup.

Reply:
We can do it by setting env vars for the different deployments but it requires manual work from an SRE. To have split.io support it we'd need to do some hacky coding in the MM server since only the cluster leader syncs the flag right now and community, -release and -daily need to all be in the same cluster.

Reply:
Any idea of how else we might be able to do that @crspeller ?

Reply:
No. Environment variables is the escape hatch for situations like community. If we want strange feature flag configurations that's how we do it.